<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReptateMainWindow</class>
 <widget class="QMainWindow" name="ReptateMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RepTate</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainWindow.qrc">
    <normaloff>:/Icons/Images/Reptate64.ico</normaloff>:/Icons/Images/Reptate64.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="ApplicationtabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMWD"/>
   <addaction name="actionTTS"/>
   <addaction name="actionLVE"/>
   <addaction name="actionNLVE"/>
   <addaction name="actionGt"/>
   <addaction name="actionCreep"/>
   <addaction name="actionSANS"/>
   <addaction name="separator"/>
   <addaction name="actionMaterials_Database"/>
   <addaction name="separator"/>
   <addaction name="actionAbout_Qt"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/Exit.ico</normaloff>:/Icons/Images/Exit.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Exit Reptate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionTTS">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/new_icons/icons8-TTS.png</normaloff>:/Icons/Images/new_icons/icons8-TTS.png</iconset>
   </property>
   <property name="text">
    <string>TTS</string>
   </property>
   <property name="toolTip">
    <string>Time-Temperature Superposition</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionLVE">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/new_icons/icons8-LVE.png</normaloff>:/Icons/Images/new_icons/icons8-LVE.png</iconset>
   </property>
   <property name="text">
    <string>LVE</string>
   </property>
   <property name="toolTip">
    <string>Linear Viscoelasticity</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionNLVE">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/new_icons/icons8-NLVE.png</normaloff>:/Icons/Images/new_icons/icons8-NLVE.png</iconset>
   </property>
   <property name="text">
    <string>NLVE</string>
   </property>
   <property name="toolTip">
    <string>Start up of shear/extension</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="actionGt">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/new_icons/icons8-Gt.png</normaloff>:/Icons/Images/new_icons/icons8-Gt.png</iconset>
   </property>
   <property name="text">
    <string>Gt</string>
   </property>
   <property name="toolTip">
    <string>Stress relaxation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+6</string>
   </property>
  </action>
  <action name="actionMaterials_Database">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/Library.ico</normaloff>:/Icons/Images/Library.ico</iconset>
   </property>
   <property name="text">
    <string>Materials Database</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/Reptate.ico</normaloff>:/Icons/Images/Reptate.ico</iconset>
   </property>
   <property name="text">
    <string> About Reptate</string>
   </property>
  </action>
  <action name="actionMWD">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/new_icons/icons8-MWD.png</normaloff>:/Icons/Images/new_icons/icons8-MWD.png</iconset>
   </property>
   <property name="text">
    <string>MWD</string>
   </property>
   <property name="toolTip">
    <string>Molecular Weight Distribution</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionCreep">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/new_icons/icons8-Creep.png</normaloff>:/Icons/Images/new_icons/icons8-Creep.png</iconset>
   </property>
   <property name="text">
    <string>Creep</string>
   </property>
   <property name="toolTip">
    <string>Creep experiments</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+7</string>
   </property>
  </action>
  <action name="actionSANS">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/new_icons/icons8-SANS.png</normaloff>:/Icons/Images/new_icons/icons8-SANS.png</iconset>
   </property>
   <property name="text">
    <string>SANS</string>
   </property>
   <property name="toolTip">
    <string>Small Angle Neutron Scattering</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+8</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/qt.ico</normaloff>:/Icons/Images/qt.ico</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Icons/Images/Help.ico</normaloff>:/Icons/Images/Help.ico</iconset>
   </property>
   <property name="text">
    <string>Contents</string>
   </property>
   <property name="toolTip">
    <string>Main Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>ReptateMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
